modules/unauthenticated-users-cluster-role-binding-con.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * authentication/using-rbac.adoc
+// * post_installation_configuration/preparing-for-users.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="unauthenticated-users-cluster-role-bindings-concept_{context}"]
+= `ClusterRoleBindings` for unauthenticated groups
+
+{product-title} {product-version} does not allow unauthenticated groups access to `ClusterRoleBindings` for security.
+
+In previous {product-title} versions unauthenticated groups were included in accessing `ClusterRoleBindings`. Clusters depolyed with previous {product-title} versions will still behave this way.
+
+In specific use cases unauthenticated users may need access to ClusterRoleBindings. For example, the `system:webhook` role allows unauthenticated users to trigger builds from external systems like GitHub, GitLab, and Bitbucket, which is important to BuildConfigs processes.
```

Comment:
* may -> might per ISG
* The last part of the second sentence is running on a little. I wonder if it's critical, or if it could be left off? If you do keep it, I think you need to either write it as "build config processes" or "`BuildConfig` processes"

```suggestion
In specific use cases, unauthenticated users might need access to cluster role bindings. For example, the `system:webhook` role allows unauthenticated users to trigger builds from external systems such as GitHub, GitLab, and Bitbucket.
```